crate/pkcs11/provider/src/tests.rs

```diff
@@ -47,13 +47,16 @@ fn initialize_backend() -> Result<CkmsBackend, Pkcs11Error> {
             .await
             .unwrap();
```

Comment:
it might panic

Reply:
its true that are tests. so maybe panicking is not a big deal.

Reply:
as discussed together, `unwrap()` are allowed only in tests and benches.